### DISCUSSION TITLE
React vs. Vue

### DISCUSSION THREAD

**Discussion by alfonso-ramos** [Launch-X-Latam/MisionFrontEnd#55] [Polls]
Un framework y librería muy buenos y bastante populares, pero nuestro MC solo nos podrá guiar por uno, cual erigirían?

**Comment by Ireneripo-LaunchX** [comment]
Hola, ya existe esta encuesta [acá](https://github.com/Launch-X-Latam/MisionFrontEnd/discussions/31#discussion-4424287).

**Reply by NestorMyrie** [reply]
se han creaco como 20 encuestas con esta pregunta

**Reply by mikaelzen** [reply]
> se han creaco como 20 encuestas con esta pregunta

Y ninguna es la oficial xD
